pkg/lib/machine-info.js

```diff
@@ -36,6 +36,12 @@ export function cpu_ram_info(address) {
                     var total_kb = match && parseInt(match[1], 10);
                     if (total_kb)
                         info.memory = total_kb * 1024;
+
+                    var free_match = text.match(/MemFree:[^0-9]*([0-9]+) [kK]B/);
```

Comment:
It seems to me that we should rather show [MemAvailable](https://github.com/torvalds/linux/blob/master/Documentation/filesystems/proc.txt#L924), to not completely write off memory that is being used as cache. `MemFree` really is misleading, as a good operating system uses all the memory that there is for caching (it would otherwise be completely useless).

@garrett, I think you most recently deep-dived into that. Do you still remember?

Reply:
Agreed.

It should be:

`(total - available)` vs `total`

As available is the value that really matters, and we're showing the amount of memory used versus total.

Details about all of that @ https://www.linuxatemyram.com/

(The buffers/cache, shared, and "free" matter to some people who know how the kernel works and how Linux tries to keep file accesses in RAM whenever possible, to actually use the RAM... but available is the amount that's easy & painless to discard for active usage... so that's the part that _really_ matters.)